src/workerd/util/symbolizer.c++

```diff
@@ -0,0 +1,133 @@
+#include <cstdint>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/wait.h>
+#include <unistd.h>
+#include <kj/common.h>
+#include <kj/debug.h>
+#include <kj/exception.h>
+#include "dlfcn.h"
+
+// Stack trace symbolizer. To use link this source to the binary.
+// Current implementation shells out to $LLVM_SYMBOLIZER if it is defined.
+
+namespace kj {
+
+struct Subprocess {
```

Comment:
It's easy to imagine KJ growing its own subprocess API someday, which could have symbol name conflicts with the one defined here. So this probably shouldn't be in the KJ namespace. Only `stringifyStackTrace` needs to be declared inside `namespace kj`...

Reply:
it is in the top comment: `-c dbg --test_env=LLVM_SYMBOLIZER=llvm-symbolizer-15`

Reply:
we can always make things default later